apps/system/style/chrome/chrome.css

```diff
@@ -149,7 +149,7 @@
   background-color: rgba(204, 204, 204, 0.2);
 }
 
-.chrome .controls .urlbar .title {
+.chrome-combined .title {
```

Comment:
I tried to reduce the specificity of some selectors so that I could override them with more simple selectors